### PR TITLE
Prepare ML Edit layout

### PR DESCRIPTION
**Pull request by mfrachet**
### What does it do?

- Prepare the ML edit dialog layout

### Why is it needed?

🙄 

### How to test it?

- Go to the media library in the left side menu
- Add a new asset
- After going through the addition process and being back on the listview, click the edit pen on the media card
- Assert that the UI is valid

### Screenshot

Screenshot 2021-10-04 at 14 17 07

### CODE REVIEW

File: packages/core/upload/admin/src/components/EditAssetDialog/index.js [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+      <ModalHeader>
+        <ButtonText textColor="neutral800" as="h2" id="title">
+          {formatMessage({ id: getTrad('modal.edit.title'), defaultMessage: 'Details' })}
+        </ButtonText>
+      </ModalHeader>
+      <ModalBody>
+        <Grid gap={4}>
+          <GridItem xs={12} col={6}>
+            <PreviewBox>
+              <img src={prefixFileUrlWithBackendUrl(asset.url)} alt={asset.name} />
```
Comment (HichamELBSI) [inline packages/core/upload/admin/src/components/EditAssetDialog/index.js:40]: It wont work for file & videos. Are we expecting to see the video preview in this PR?

**Review by HichamELBSI — COMMENTED** [review]
The grid is not very suitable when one or two media are displayed. Maybe we can add a min width or something to fix that 🤷🏽‍♂️ 

Screenshot 2021-10-04 at 14 37 45

File: packages/core/upload/admin/src/components/EditAssetDialog/index.js (hunk 1 above)
Comment (mfrachet) [reply packages/core/upload/admin/src/components/EditAssetDialog/index.js:40]: Not in this PR, I don't really what should be loaded for now 😅

**Comment by mfrachet** [comment]
> The grid is not very suitable when one or two media are displayed. Maybe we can add a min-width or something to fix that 🤷🏽‍♂️

Totally agree. We'll need to figure out what to do in this scenario. We could eventually min-max but it will create a weird effect on screen. Will wait for some QAs and ajustment and try to make this thing work asap :)

**Review by soupette — APPROVED** [review]
LGTM

**Pull request merged**